src/app.js

```diff
@@ -1 +1,6 @@
 'use strict';
+
+const { terminal } = require('./terminal');
+const { handleUserAnswer } = require('./handleUserAnswer');
+
```

Comment:
How about adding a welcome message that would be displayed at the start of your game, e.g: "Welcome to bulls and cows".